Resources/ServerInfo/Guidebook/DeltaV/Rules/GameRules/2_Metagaming.xml

```diff
@@ -1,14 +1,131 @@
+
+  ## High-Risk Items
+
+  The idea that certain High-Risk Items are desired specifically by the Syndicate is shielded. Crew should protect High-Risk items due to their status as valuable items, not because they are potential steal objectives.
+
+  The revealing condition for this shield is any of the following:
+   - being a traitor
+   - being told of an item’s status by a traitor
+
+  ## Nuclear Operatives
+
+  The existence of Nuclear Operatives beyond a myth that no one would act on is shielded.
+
+  The fact that the nuke disk must be protected and could be used by a bad actor to try to destroy the station is not shielded.
+
+  The revealing condition for this shield is any of the following:
+  - discovering a blood red hardsuit
+  - discovering a nuclear operative's shuttle
+  - an operative name
+  - a War Ops announcement
+  - being a nuclear operative
+
+  ## Implants
+
+  The existence of implants used by the Syndicate is shielded. Implant checks cannot be performed unless there is adequate evidence or an authorized warrant, as outlined within Standard Operating Procedures.
+
+  The revealing condition for this shield is any of the following:
+  - discovering a non-NT implanter, used or unused
+  - discovering a non-NT implant box
+  - discovering use of a non-NT implant by anyone
+  - experiencing a situation where absolutely no other explanation is possible
+  - discovering an unlocked uplink
+
+  ## Chameleon Items
+
+  Chameleon items are shielded.
+
+  Being suspicious of an item being fake or stolen is not shielded, but testing items or calling them chameleon is covered by this shield. Testing clothing for chameleon status is always prohibited; items that are clearly chameleon can, however, be confiscated by Security.
+
+  The revealing condition for this shield is any of the following:
+  - seeing someone else cause any chameleon item to change
+  - finding holographic nanomachine fibers
+  - experiencing a situation where absolutely no other explanation is possible
+  - discovering an unlocked uplink
+
+  ## Stealth Items
+
+  The fact that an item can be something other than what its visual appearance and examine description indicate is shielded.
+
+  This shield protects stealth items, including protecting them from being tested.
+
+  The revealing condition for this shield is any of the following:
+  - seeing the item behave differently than the expected behavior for the item
+  - seeing the item used for its hidden purpose
+  - experiencing a situation where absolutely no other explanation is possible
+  - discovering an unlocked uplink
+
+  ## Bad Actor Objectives
+
+  The idea that Syndicate agents, or other bad actors, have certain goals they have been tasked with accomplishing on the station, ranging from sabotage, to theft, to murder is shielded.
+
+  The revealing condition for this shield is any of the following:
+   - being a Traitor, Space Ninja, Space Dragon, or Paradox Anomaly
```

Comment:
there are more antags than this :trollface:

Reply:
Got this too 👍